tmt/steps/provision/connect.py

```diff
@@ -64,6 +64,7 @@ def default(self, option, default=None):
         if option == 'user':
             return 'root'
         # No other defaults available
+
```

Comment:
Nitpick but a blank line here is added in the patch (perhaps a leftover from a previous iteration).